.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing to Haaukins 
+
+If you wish to make contribution to Haaukins platform, you are very welcome to do it, any contribution would be beneficial. 
+
+## Pull requests are always welcome
+
+We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. Not sure if that typo is worth a pull request? Do it! We will appreciate it.
+
+If your pull request is not accepted on the first try, don't be discouraged! If there's a problem with the implementation, hopefully you received feedback on what to improve.
+
+## Conventions 
+
+Fork the repo and make your changes on your fork in a feature branch. Haaukins developments is based on [Git Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+
+- For bug fixes and feature requests, the name of branch that you will work on should follow the convention [XXXX]-something, XXXX refers to issue number. 
+
+- There should be a naming convention (function names and variable names should follow defined pattern such as getName(private) or GetName (public).
+
+- Having clean formatted codes are always easy to review, hence the changes on branch require to have `gofmt` to format the code. 
+
+- Commit messages should start with capitalized letter and follow short summary of the changes. Commits which are fixing or closing an issue should include reference to it, such as `Closes #[XXXX]` or `Fixes #[XXXX]` 
+
+- Before submit your changes and having a pull request, make sure that all tests passed. 
+
+- Have descriptive comments on changes.
+
+## For Maintainers
+
+### Code Review Process 
+
+- All changes to the master branch must be through Pull Requests, and all PRs must be approved by at least one maintainer, who must be different from the creator of the PR. 
+
+- Pull requests should be made into version branch instead of master, since latest version will be identical with master branch, having pull request to version branch will bump version according to changes.
+
```

Comment:
Should the PR made by a contributor point to the version branch or should the maintainer merge the PR into the version branch? (I guess only the maintainer knows this)

Reply:
I am a little bit confused for the initial question, 

> Should the PR made by a contributor so it points to the right version branch or should the maintainer merge the PR into the version branch?

Making PR and merging the PR are two different things, so a contributor will open a pull request to version branch which is identical to latest version. For example, let's say, 1.6.3 is latest version branch (identical with master), and ideally `( - automated release script has created the next patched version called 1.6.4 (it is identical to 1.6.3 )- )` so when 1.6.4 branch is created, next developments should take place based on 1.6.3, and when pull requests created the base target should be 1.6.4, so 1.6.4 will contain the changes which are made from 1.6.3.

Reply:
Sorry if my question was not clear, but I think you got it anyway. At least you answered what I wanted to ask :)

I don't think the point you just made is clear from CONTRIBUTING.
Do you mind having a second look at it, if you indeed think it is.
Modify if relevant, or indicate if you think it's fine - then I'm happy too :+1:

Reply:
nvm. You already did that..